package.json

```diff
@@ -93,6 +93,7 @@
     ]
   },
   "dependencies": {
+    "@firebase/logger": "^0.3.4",
```

Comment:
Hmm this is adding new dep to the CLI. Is this intentional or should this be a dev dep?